### PR TITLE
It begins

### PR DESCRIPTION
**Pull request by hoo29**

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-  siteMetadata: {
-    title: `Gatsby Typescript Starter`,
-  },
-  plugins: [
-    `gatsby-plugin-react-helmet`,
-    // Add typescript stack into webpack
-    `gatsby-plugin-typescript`
-  ],
+    siteMetadata: {
+        title: `Gatsby Typescript Starter`,
```
Comment (TerminalStar) [inline gatsby-config.js]: Need to change this, yo

File: src/layouts/index.scss [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  /* options
+  $base-grid: true,
+  $margin-grid: true,
+  $padding-grid: true,
+  $block-grid: true,
+  $collapse: true,
+  $offset: true,
+  $vertical-grid: true,
+  $frame-grid: true
+  */
```
Comment (TerminalStar) [inline src/layouts/index.scss]: plz get rid of this at some point

File: src/layouts/index.tsx [hunk 3]
```diff
@@ -43,25 +17,44 @@ interface DefaultLayoutProps extends React.HTMLProps<HTMLDivElement> {
...
-                    {this.props.children()}
+            <div id='outer-container'>
+                <div className='grid-x'>
+                    <div className='cell medium-2'>
+                        <Menu pageWrapId={'page-wrap'} outerContainerId={'outer-container'}>
+                            <a id='home' className='menu-item' href='/'>Home</a>
+                            <a id='about' className='menu-item menu-item--about' href='/about'>About</a>
+                            <a id='about' className='menu-item menu-item--sub' href='/about'>Consultants</a>
+                            <a id='about' className='menu-item menu-item--sub' href='/about'>Projects</a>
+                            <a id='about' className='menu-item menu-item--sub' href='/about'>Testimonials</a>
```
Comment (TerminalStar) [inline src/layouts/index.tsx]: these all have the wrong ids

File: tslint.json [hunk 4]
```diff
@@ -21,7 +21,8 @@
         "quotemark": [
             true,
             "single"
-        ]
+        ],
+        "max-line-length": 300
```
Comment (TerminalStar) [inline tslint.json:25]: lol do we want to keep this?

Comment (hoo29) [reply tslint.json:25]: yes

**Review by TerminalStar — APPROVED** [review]
🏅

**Pull request merged**